lcls_tools/common/devices/screen.py

```diff
@@ -73,7 +73,7 @@ def image(self) -> np.ndarray:
         the camera associated with this screen
         """
         return self.controls_information.PVs.image.get(as_numpy=True).reshape(
-            self.n_rows, self.n_columns
+            self.n_columns, self.n_rows
```

Comment:
How did this mistake get caught? I think I'm remembering this might not be the case for all cameras. Will approve for now and check with Courtney.